crates/albedo_rtx/src/passes/accumulation.rs

```diff
@@ -183,21 +170,24 @@ impl AccumulationPass {
                     resource: wgpu::BindingResource::Sampler(sampler),
                 },
             ],
-        })
+        });
+        [scene_bind_group, bind_group]
     }
 
     pub fn dispatch(
         &self,
         encoder: &mut wgpu::CommandEncoder,
-        frame_bind_groups: &wgpu::BindGroup,
+        bind_groups: &[wgpu::BindGroup; 2],
```

Comment:
Does that work with a slice of size 4? I would probably allow any size and `panic!` if it's less than 2. Maybe someone will store all the vind groups in a Vec<> I dont know